Lemons/main.cpp

```diff
@@ -0,0 +1,97 @@
+//
+//  main.cpp
+//  lemonade
+//
+//  Created by Herman Sollerman on 27/03/16.
+//  Copyright © 2016 Herman Sollerman. All rights reserved.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int number_of_cases, number_of_days, lemon_per_cup, sugar_per_cup, cups_sold, lemon_price, total_cost=0, sugar_price, j, k;
+    FILE *input;
+
+    input = stdin;
+    fscanf(input, "%d", &number_of_cases);
+    
+    for (k=1; k <= number_of_cases; k++)
+    {
+        fscanf(input, "%d %d %d", &number_of_days, &lemon_per_cup, &sugar_per_cup);
+        int total_cups [number_of_days];
+        int lprices [number_of_days];
+        int sprices [number_of_days];
+        
+        int n=0;
+        
+        for (j=1; j<=number_of_days; j++)
```

Comment:
Med 
for (int j = 0; j < number_of_days; ++j) 
så skulle du kunna skippa n här och köra med j istället.